src/activities/binary_bulb/LightBulb.qml

```diff
@@ -0,0 +1,50 @@
+import QtQuick 2.6
+
+Image {
+	id: bulb
+	anchors.verticalCenter: parent.verticalCenter
+	width: 50
+	height: 100
+	source: "resource/bubl.svg"
+	state: "off"
+
+	property string t: ""
+
+	MouseArea {
```

Comment:
so the other MouseArea in the activity is not necessary, you should handle everything here.